index.html

```diff
@@ -202,7 +201,7 @@ <h1 class="professional-h1">Professional Art<br />Printing Data</h1>
           <li class="listons second">bootstrap</li>
           <li class="listons third">ruby</li>
         </ul>
-        <button class="liston-button bb1">See Project</button>
+        <button class="liston-button bb1 cproject">See Project</button>
```

Comment:
- [ ] Add more padding for your `li` elements in your popup.
- [ ] Align the bottom from the main image with the "see live" and the "see source" buttons.
- [ ] Add the right font-family for all your popup.
- [ ] Add the right "X" close icon according to the Figma design you selected.
- [ ] Add blur background when you are in your popup.

Figma design | Your design

![image](https://user-images.githubusercontent.com/60164239/131805631-c82c833a-7425-4bca-b878-5f2757e6b76a.png)